sky/data/storage.py

```diff
@@ -996,8 +1019,13 @@ def upload(self):
                 f'Upload failed for store {self.name}') from e
 
     def delete(self) -> None:
-        self._delete_s3_bucket(self.name)
-        logger.info(f'{colorama.Fore.GREEN}Deleted S3 bucket {self.name}.'
+        deleted_by_skypilot = self._delete_s3_bucket(self.name)
+        if deleted_by_skypilot:
+            msg_str = f'Deleted S3 bucket {self.name}.'
```

Comment:
While testing, I sky launched an instance while creating and mounting a new bucket called `s3-doyoung-delete`. Then, externally deleted from AWS console. Finally, running `sky storage delete s3-doyoung-delete` returned this message while it should return L1026 instead.